components/Wallet/Wallet.tsx

```diff
@@ -27,8 +26,7 @@ export function Wallet() {
   }, [connectedWallets, wallet]);
 
   useEffect(() => {
-    if (!connect) return;
-
+    if (!connect || connectedWallets?.length > 0) return;
```

Comment:
This was a bug. It would run the automatic connection logic even when a connected wallet existed, which would show an error